ouroboros-consensus-cardano/test/cardano-test/Test/ThreadNet/ShelleyAllegra.hs

```diff
@@ -163,7 +154,10 @@ tests = testGroup "ShelleyAllegra ThreadNet" $
       adjustTestMode :: TestTree -> TestEnv -> TestTree
       adjustTestMode tree = \case
         Nightly -> tree
-        _       -> adjustOption oneTenthTestCount tree
+        _       ->
+          -- These tests are slow, so we settle for running fewer of them in this test
```

Comment:
I see this comment repeated multiple times. Perhaps we can mention somewhere else why one would want to use `adjustQCTests` (using this comment), and use line 160 as an example. Then one might not need to add these comments. What do you think?

Reply:
I was also finding it a bit silly to move the same comment over and over again. I suppose a good place could be the documentation of `adjustQuickCheckTests`? Would that be sufficient? Or would you imagine something else elsewhere?

Reply:
Yes, also think `adjustQuickCheckTests` is a good place :+1:

Reply:
What do you think of 8413f17b9?

Reply:
Looks good. Consider adding an example of how to use it, eg: 

```haskell
-- > adjustQuickCheckTests (`div` 10) 
-- 
-- will reduce the default number of tests by 10
```

Or something similar. But it's also fine as it is.

Reply:
Great idea! Done in 6b55de0fb.